tools/ci/travis-install.sh

```diff
@@ -26,5 +26,10 @@ python tools/ci/travis-install.py
 # Install Python API in editable mode
 pip install -e .[test]
 
+# conditionally install vtk
+if [ $TRAVIS_PYTHON_VERSION -ne 3.7 ]; then
```

Comment:
Just noticed that this doesn't actually work (primarily because `-ne` is an integer operator) and Travis is just skipping the vtk tests. I would change this to (combining with previous line):
```bash
if [[ $TRAVIS_PYTHON_VERISON == 3.7 ]]; then
    pip install -e .[test]
else
    pip install -e .[test,vtk]
fi
```

Reply:
Whoops. Nice catch. Thanks!